public/locales/auset-te.json

```diff
@@ -0,0 +1,426 @@
+      "setupTitle": "లిజార్డ్ కల్ట్‌ని సెటప్ చేయండి",
+      "setup": "<i>1.</i> శత్రు భూభాగాలకు ఆనుకొని లేని స్వదేశీ క్లియరింగ్‌ను ఎంచుకోండి.<br/><i>2.</i> 4 <strong>యోధులు</strong> మరియు 1 సరిపోలే <strong> ఉంచండి తోట</strong>మీ స్వదేశంలో. 3 <strong>యోధులను</strong> దాని ప్రక్కనే ఉన్న క్లియరింగ్‌లలో వీలైనంత సమానంగా ఉంచండి.<br/><i>3.</i> మీ అకోలైట్స్ బాక్స్‌లో 2 <strong>యోధులను</strong> ఉంచండి.<br /><i>4.</i> మీ గార్డెన్స్ ట్రాక్‌లను <strong>గార్డెన్స్</strong>తో పూరించండి, మ్యాప్‌లో గార్డెన్‌కి ఎడమవైపున ఉన్న స్థలం మినహా.<br/><i>5.</i> ఉంచండి మీ <strong>అవుట్‌కాస్ట్ మార్కర్</strong> అవుట్‌కాస్ట్ బాక్స్‌లోని ఏదైనా స్థలంలో దాని అవుట్‌కాస్ట్ వైపు ఉంటుంది."
+    },
+    "duchy": {
+      "name": "భూగర్భ ప్రదేశం(అండర్గ్రౌండ్ డచి)",
+      "summaryTitle": "డచి గా ఆడటం",
+      "summary": "<p>అండర్‌గ్రౌండ్ డచీగా, మీరు వుడ్‌ల్యాండ్‌లోని విదేశీ జీవులకు సబ్జెక్ట్‌లుగా మెరుగ్గా ఉంటారని చూపించాలనుకుంటున్నారు. ప్రతిసారి మీరు మీ <strong>మంత్రులలో ఒకరి</strong>ని శాంతింపజేసేందుకు మరియు పైన ఉన్న భూమిని ఏకీకృతం చేయడానికి, మీరు పాయింట్లను స్కోర్ చేస్తారు. మీరు ఎంత ఎక్కువ మంత్రిగా ఉన్నారో, మీరు అంత ఎక్కువ పాయింట్లను స్కోర్ చేస్తారు.</p><p>ఒక మంత్రిని తిప్పికొట్టడానికి మీరు వుడ్‌ల్యాండ్ వేర్పాటువాదులలో మీ మద్దతును సూచించే కార్డ్‌లను తప్పనిసరిగా బహిర్గతం చేయాలి. మంత్రి ర్యాంక్ ఎంత ఎక్కువ ఉంటే, మీరు మరిన్ని కార్డులను బహిర్గతం చేయాలి. అయినప్పటికీ, మీరు ఎన్ని డచీ పీస్‌లతోనైనా క్లియరింగ్‌లకు సరిపోయే కార్డ్‌లను మాత్రమే బహిర్గతం చేయగలరు, ఇది తిరుగుతున్న గందరగోళంలో మీ పట్టును చూపుతుంది. ఊగిసలాడే ప్రతి మంత్రి మీకు ప్రతి మలుపులో అదనపు చర్యను అందిస్తారు. కొంతమంది మంత్రులు వుడ్‌ల్యాండ్‌లో మీ పెట్టుబడులను ప్రచారం చేయడం ద్వారా విజయ పాయింట్లను స్కోర్ చేయడానికి మిమ్మల్ని అనుమతిస్తారు.</p><p>మీ పుట్టుమచ్చలు <strong>ది బురో</strong>లో బాగా రక్షించబడ్డాయి, మీరు మాత్రమే ప్రవేశించగలరు మరియు మీరు ఎల్లప్పుడూ పాలించగలరు . బురో నుండి, మీరు వుడ్‌ల్యాండ్ అంతటా తవ్విన ఏదైనా <strong>సొరంగం</strong>కి వెళ్లవచ్చు. మీరు క్లియరింగ్‌లను శాంతింపజేసి, నియమాన్ని ఏర్పరచుకున్న తర్వాత, మీ ప్రయత్నానికి మరిన్ని జీవులను ఆకర్షించడానికి మీరు <strong>సిటాడెల్స్</strong> మరియు <strong>మార్కెట్‌లను</strong> నిర్మించవచ్చు. అయితే, మీరు భవనాలను కోల్పోయినప్పుడల్లా, అత్యున్నత స్థాయి మంత్రితో అధికారాన్ని కోల్పోయి, యాదృచ్ఛిక కార్డ్‌ని విస్మరించడం ద్వారా మీరు <strong>వైఫల్య ధర</strong>ని తప్పనిసరిగా చెల్లించాలి.</p>",
+      "setupTitle": "భూగర్భ డచీని ఏర్పాటు చేయండి",
+      "setup": "1 </strong> మీ స్వదేశంలో. 5 <strong>యోధులను</strong> దాని ప్రక్కనే ఉన్న క్లియరింగ్‌ల మధ్య వీలైనంత సమానంగా ఉంచండి.<br/><i>3.</i> మీ <strong>బురో బోర్డ్‌ను</strong> మ్యాప్ దగ్గర ఉంచండి. మీ <strong>సిటాడెల్స్</strong> మరియు <strong>మార్కెట్లు</strong>తో మీ భవనాల ట్రాక్‌లను పూరించండి. మీ 9 <strong>మంత్రి కార్డ్‌లను</strong> మీ అస్థిర మంత్రుల పైల్‌పై ఉంచండి. మీ 9 <strong>కిరీటాలను</strong> విజయ పాయింట్లను చూపుతున్న చతురస్రాకార ఖాళీలపై ఉంచండి."
+    },
+    "eyrie": {
+      "name": "ఐరీ రాజవంశాలు(ఐరీ డైనసైటిస్)",
```

Comment:
I think the phonetic translation is a nice touch, however it makes the faction name too long to fit neatly in the interface. I would suggest just using the faction name from the official Leder games translation, or if that doesn't exist use just the literal translation without the phonetic one.